File: lektionen/javascript/Kata/k6_which_are_in.js
function inArray(array1,array2){
  let arr = [];
  let arr2 = [];
  array1.forEach(str => {arr = arr.concat(array2.join(' ').match(str));});
  arr.forEach(x => {
    if (x != null) {
      arr2.push(x);
    }
  });
  return arr2.sort();
}

File: lektionen/python/Kata/k6_which_are_in.py
def in_array(array1, array2):
  arr = []
  for e in array1:
    for i in array2:
      if e in i and e not in arr:
        arr.append(e)
  return sorted(arr)

print(in_array(["live", "arp", "strong"], ["lively", "alive", "harp", "sharp", "armstrong"]))